app/javascript/packs/homePageFeed.jsx

```diff
@@ -92,9 +89,10 @@ export const renderFeed = timeFrame => {
         // 3. Rest of the stories for the feed
         return (
           <div>
-            <FeaturedArticle
+            <Article
               {...commonProps}
               article={featuredStory}
+              isFeatured={true}
```

Comment:
This fixes a lint issue and since this value is always true, you can use the shorthand below. For more info see https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md.

```suggestion
              isFeatured
```